tests/lib/UI/Config/Provider/Module/DamWidgetTest.php

```diff
@@ -141,29 +148,60 @@ public function testGetConfigThrowInvalidSearchEngine(): void
     /**
      * @return iterable<array{
      *     TDamWidgetConfig,
-     *     TRepositoryConfig
+     *     TRepositoryConfig,
+     *     array<array{string|array<string>|ContentType}>,
+     *     array<array{string|array<string>}>,
```

Comment:
From what I see here, you can be more specific.

```suggestion
     *     array<array{string, array{}, ContentType}>,
     *     array<array{string, array{field: string}}>,
```

Reply:
Fixed: https://github.com/ibexa/admin-ui/pull/1236/commits/2b85b8915c7d73dad07c0b554da97b1b3e6371db